### PR TITLE
Windows: implement native bluetooth support !

### PR DESCRIPTION
**Pull request by gpotter2**
This PR:
- add bluetooth sending/recieving sockets support for windows. It's not available in vanilla python, so we need to perform raw calls to C functions. Most of the code is extracted from a compatibility part of the pybluez project, and i added manualy x64 compatibility.
- I tested it and included screenshots in the doc.
- I have added the Visual Studio project in a zipped file

### CODE REVIEW

**Comment by gpotter2** [comment]
This update adds many features to the windows install.
I will do the same for linux in a near future

File: scapy/arch/windows/bluetooth_socket.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+## This file is part of Scapy
```
Comment (guedou) [inline scapy/arch/windows/bluetooth_socket.py]: Could you use a linter on this file ? I have seen some PEP08 errors.

**Comment by guedou** [comment]
Why are you including a zip file ?

**Comment by gpotter2** [comment]
I actually am not sure about this PR anymore...

It's working great, but it packs some `.pyd` files which basically are executables :/
In an opensource project such as scapy, because those are binaries (to be exportable) they could easily be virused with malicious content...

**Comment by guedou** [comment]
I agree that it is quite intrusive. I think that you could make theses changes leave in an independent repository that uses Scapy a Python module.

I will close this PR for now. You can reopen it if you make up your mind =)

**Pull request closed without merging**